dali/operators/reader/loader/numpy_loader.cc

```diff
@@ -237,7 +237,7 @@ void NumpyLoader::ReadSample(NumpyFileWrapper& target) {
     target.data.SetMeta(meta);
     target.data.set_type(TypeInfo::Create<uint8_t>());
     target.data.Resize({0});
-    target.filename = "";
+    target.filename.clear();
```

Comment:
A personal opinion: hile possibly clear is faster, I think that `= ""`.

Reply:
FYI: It's the custom in this project that the it's the reviewer that marks a conversation/issue as resolved.